modules/1_bastion/bastion.tf

```diff
@@ -74,9 +74,40 @@ locals {
   }
 }
 
+resource "null_resource" "bastion_fips" {
+  count = var.fips_compliant ? local.bastion_count : 0
+
+  connection {
+    type         = "ssh"
+    user         = var.rhel_username
+    host         = openstack_compute_instance_v2.bastion[count.index].access_ip_v4
+    private_key  = var.private_key
+    agent        = var.ssh_agent
+    timeout      = "${var.connection_timeout}m"
+    bastion_host = var.jump_host
+  }
+
+  provisioner "remote-exec" {
+    inline = [<<EOF
+fips-mode-setup --enable
+systemctl reboot
```

Comment:
Can make it run as sudo user?
Also wondering if there is a better way to reboot the instance.

Reply:
Switched to sudoer

I thought this as the first null_resource provider it provided the least risk to do `systemctl reboot`.  Do you have ideas for alternatives?